### PR TITLE
Show projects list 

### PR DESCRIPTION
**Pull request by vladhrapov**
Complete task show projects list in menu, get projects list from DB, send project item to DB, create form for user

### CODE REVIEW

File: client/js/app/main/modules/menu/index.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 var angular = require('angular');
 
 var menuModule = angular.module('menuModule', [
-    require('./projectList')
+    require('./projectList'),
+    require('./createProject')
 ])
```
Comment (borys-ihnatyev) [inline client/js/app/main/modules/menu/index.js:5]: createProject module should be in main/modules folder and be exported from main/modules/index.js

File: client/js/app/main/modules/menu/createProject/createProject.config.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/**
+ * @ngInject
+ */
+module.exports = function ($stateProvider) {
+    $stateProvider.state('main.createProject', {
+        url: '/project/create',
```
Comment (borys-ihnatyev) [inline client/js/app/main/modules/menu/createProject/createProject.config.js]: For generic url consistency - url: 'project**s**/create'

File: client/js/app/main/modules/createProject/createProject.service.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+module.exports = function($http){
+
+    this.sendData = function (project) {
```
Comment (kucherenko) [inline client/js/app/main/modules/createProject/createProject.service.js]: make name of this method and file more clear, sendData in createService does not explain anything

File: client/js/app/main/modules/createProject/createProject.service.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+        console.log('name: ' + project.name + ' color: ' + project.color);
+        $http.post('/api/projects/add', { name: project.name, color: project.color });
```
Comment (kucherenko) [inline client/js/app/main/modules/createProject/createProject.service.js]: Also we can move this call to projectService

**Pull request closed without merging**